lib/common/tests/DevModeTest.php

```diff
@@ -34,7 +34,7 @@ public function dataHasExemptionForNode()
     {
         return [
             [ '<html><body id="node_to_test"><div data-ampdevmode></div></body></html>', false ],
-            [ '<html><body><div id="node_to_test" data-ampdevmode></div></body></html>', true ],
+            [ '<html data-ampdevmode><body><div id="node_to_test" data-ampdevmode></div></body></html>', true ],
```

Comment:
Two more test cases are needed, namely:

1. On `<html id="node_to_test">` when `data-ampdevmode` is initially present, `DevMode::hasExemptionForNode($node)` should be `true`.
1. On `<html id="node_to_test">` when `data-ampdevmode` is not initially present (added after document created), `DevMode::hasExemptionForNode($node)` should be `false`.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4440/commits/3081a0a5de6e45c3d01ea0c9059f9914dc37128b.